macos/Sources/Ghostty/SurfaceView_AppKit.swift

```diff
@@ -876,7 +879,7 @@ extension Ghostty {
             var x = event.scrollingDeltaX
             var y = event.scrollingDeltaY
             let precision = event.hasPreciseScrollingDeltas
-            
+
```

Comment:
While we appreciate the cleanup of the whitespace, it should at least be confined to a separate commit to make review easier.

Reply:
No problem! Just had a little IDE format on save action happen here… I’ll get this cleaned up prior to taking the PR out of draft status